app/templates/components/forms/user-profile-form.hbs

```diff
@@ -11,6 +11,7 @@
         maxSizeInKb=10000
         aspectRatio=(array 1 1)
         helpText=(t 'We recommend using at least a 1000x1000px (1:1 ratio) image')}}
+      <br>  
```

Comment:
use `ui hidden divider`

Reply:
@mohitm15  I have observed usually in your PR , you always use your development branch to create PR and then you delete it . Like in this PR you deleted your development and here it shows `UNKNOWN REPOSITORY` it is not a good practice , as if mentors want you to change something you cant change and you need to open a new PR . This is not a good practice to follow , Please change this , Else its your wish , No problem . 
Now see you cant change your commits which @CosmicCoder96 is asking for you to do

Reply:
@CosmicCoder96 Should I close this PR and open a new as this is from unknown branch and I am unable to make changes in it.